include/aspect/postprocess/visualization/adiabat.h

```diff
@@ -59,12 +59,8 @@ namespace aspect
 
           virtual
           void
-          compute_derived_quantities_vector (const std::vector<Vector<double> >              &solution_values,
-                                             const std::vector<std::vector<Tensor<1,dim> > > &solution_gradients,
-                                             const std::vector<std::vector<Tensor<2,dim> > > &solution_hessians,
-                                             const std::vector<Point<dim> >                  &normals,
-                                             const std::vector<Point<dim> >                  &evaluation_points,
-                                             std::vector<Vector<double> >                    &computed_quantities) const;
+          evaluate_vector_field(const DataPostprocessorInputs::Vector<dim> &input_data,
+                                std::vector<Vector<double>> &computed_quantities) const;
```

Comment:
`>>` will be interpreted by older compilers as the operator `>>`. Please change it to `> >`.
Same in all other places.